### ISSUE TITLE
(MarkdownView) Support multiline math

### ISSUE THREAD

**Issue by karino2** [karino2/kotlitex#135]
Currently, we do not support multiple line of math exp like below.

```
$$
f(x) = n -1 \\
g(y) = 3+4
$$
```

Sometime I need this.

**Comment by karino2** [comment]
Here is another example:

```
$$
 -x_1+x_2 \leq 1 \\
 x_1\leq 3 \\
 x_2  \leq 2 \\
 x_1,x_2\geq 0 \\
$$
```

**Comment by karino2** [comment]
Not only markdown view, but also kotlitex itself does not support `\\`.

**Comment by karino2** [comment]
Fixed  by #163
